client/src/main/java/io/pravega/client/stream/impl/ReaderGroupState.java

```diff
@@ -37,7 +37,7 @@
  * This class encapsulates the state machine of a reader group. The class represents the full state, and each
  * of the nested classes are state transitions that can occur.
  */
-class ReaderGroupState implements Revisioned {
+public class ReaderGroupState implements Revisioned {
```

Comment:
Why are we making this public (and the methods below)?

Reply:
ReaderGroupState is part of package `io.pravega.client.stream.impl.ReaderGroupState` and it should be visible/accessible inside package `io.pravega.client.stream.notifications.notifier`